tests/integration/cases/email_checking_pypi_malware_analyzer/check_emails.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Looks like something went wrong with the copyright checker. Can you please fix the start year to 2025?